src/rust/include/rust/orchard/wallet.h

```diff
@@ -174,10 +174,12 @@ bool orchard_wallet_append_bundle_commitments(
         );
 
 /**
- * Returns the root of the wallet's note commitment tree.
+ * Returns the root of the wallet's note commitment tree by assigning to `root_ret`.
+ * Returns `true`
  */
-void orchard_wallet_commitment_tree_root(
+bool orchard_wallet_commitment_tree_root(
         const OrchardWalletPtr* wallet,
+        const size_t confirmations,
```

Comment:
Why is this called `confirmations`, when it is called `checkpoint_depth` in the Rust code?

Reply:
Bearing in mind https://github.com/zcash/zcash/pull/6549/commits/be0f0bf95b0be176177afe33268ad5dcb65d8bd9#r1209484007, this looks simply wrong. The checkpoint depth and the number of confirmations are not the same thing; they differ by 1.
```suggestion
        const size_t checkpoint_depth,
```